apps/web/src/pages/templates/components/TemplateEditorFormProvider.tsx

```diff
@@ -55,13 +61,23 @@ const makeStep = (channelType: StepTypeEnum, id: string): IStepEntity => {
       },
     }),
     ...(channelType === StepTypeEnum.DIGEST && {
-      metadata: {
+      digestMetadata: {
+        digestKey: '',
         type: DigestTypeEnum.REGULAR,
+        regular: {
+          unit: DigestUnitEnum.MINUTES,
+          amount: '5',
+          backoff: false,
+        },
       },
     }),
     ...(channelType === StepTypeEnum.DELAY && {
-      metadata: {
-        type: DigestTypeEnum.REGULAR,
+      delayMetadata: {
+        type: DelayTypeEnum.REGULAR,
+        regular: {
+          unit: DigestUnitEnum.MINUTES,
+          amount: '5',
+        },
       },
```

Comment:
the default values for the digest/delay step metadata... I'll explain about these more later